### PR TITLE
WASM Aid: Less queue capacity, don't account ASCII

### PR DESCRIPTION
**Pull request by mehmetoguzderin**
To generate web playgrounds in mirror repo, WASM build workflow and the resulting file should work fine. Removing ASCII accounting and reducing queue capacity makes it possible to run WASM object once again (I use Safari 16.3 release version), but marking as draft to discuss potential alternatives that might have less risk.

To playground in web:
```
cd wgsl
make
cd grammar
npx tree-sitter build-wasm
npx tree-sitter playground
```

### CODE REVIEW

File: wgsl/scanner.cc [hunk 1]
```diff
@@ -604,10 +603,8 @@ class Lexer {
       return false;
     }
 
-    std::stringstream ss;
```
Comment (dneto0) [inline wgsl/scanner.cc:607]: Yes, let's remove this.

File: wgsl/scanner.cc [hunk 2]
```diff
@@ -668,8 +664,8 @@ class Lexer {
 
 struct Scanner {
   struct State {
-    BitQueue<1024> lt_is_tmpl;  // Queue of disambiguated '<'
-    BitQueue<1024> gt_is_tmpl;  // Queue of disambiguated '>'
+    BitQueue<64> lt_is_tmpl;  // Queue of disambiguated '<'
```
Comment (dneto0) [inline wgsl/scanner.cc:667]: This looks fine.

**Review by dneto0 — APPROVED** [review]
Thanks!

File: wgsl/scanner.cc (hunk 1 above)
Comment (mehmetoguzderin) [reply wgsl/scanner.cc:607]: Thank you!

**Pull request merged**